Library/Formula/ploticus242.rb

```diff
@@ -0,0 +1,18 @@
+class Ploticus242 < Formula
```

Comment:
Class and filename has been updated.

/Magnus

On Fri, Dec 18, 2015 at 12:35 AM, Andrew Janke <notifications@github.com>
wrote:

> In Library/Formula/ploticus242.rb
> <https://github.com/Homebrew/homebrew/pull/46831#discussion_r47977109>:
>
> > @@ -0,0 +1,18 @@
> > +class Ploticus242 < Formula
>
> This should probably be plain Ploticus and live in file ploticus.rb.
> Homebrew doesn't bake version numbers in to formula names.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/46831/files#r47977109>.
>